internal/driver/webhtml.go

```diff
@@ -53,90 +57,187 @@ button {
   margin-top: 5px;
   margin-bottom: 5px;
 }
-#reset {
-  margin-left: 10px;
-}
 #detailtext {
   display: none;
-  position: absolute;
+  position: fixed;
+  top: 20px;
+  right: 10px;
   background-color: #ffffff;
   min-width: 160px;
-  border-top: 1px solid black;
-  box-shadow: 2px 2px 2px 0px #aaa;
+  border: 1px solid #888;
+  box-shadow: 4px 4px 4px 0px rgba(0,0,0,0.2);
   z-index: 1;
 }
-#actionbox {
-  display: none;
-  position: fixed;
-  background-color: #ffffff;
-  border: 1px solid black;
-  box-shadow: 2px 2px 2px 0px #aaa;
-  top: 20px;
-  right: 20px;
-  z-index: 1;
+#closedetails {
+  display: inline;
```

Comment:
flex is probably too different from the behavior described (the button would need to be part of some header).
However, the display: inline is still redundant as float takes over.